Extension/tools/GenerateOptionsSchema.ts

```diff
@@ -80,6 +80,17 @@ function replaceReferences(definitions: any, objects: any): any {
             objects[key] = refReplace(definitions, objects[key]);
         }
 
+        // Handle 'anyOf' with references
+        if (objects[key].hasOwnProperty('anyOf')) {
+            for (const index in objects[key].anyOf)
+            {
+                if (objects[key].anyOf[index].hasOwnProperty('$ref'))
+                {
```

Comment:
The brace styling doesn't match...is TSLint not running on this file?

Reply:
We skip the `tools` directory for tslint.

Reply:
Was there a reason for that?

Reply:
Some configuration from `tsconfig` or another config file skips it, or else we would compile and webpack the files in tools into `main.js`.